Mathlib/Algebra/Algebra/Unitization.lean

```diff
@@ -658,9 +657,6 @@ theorem algHom_ext' {φ ψ : Unitization R A →ₐ[R] C}
     φ = ψ :=
   algHom_ext'' (NonUnitalAlgHom.congr_fun h)
 
-/- porting note: this was extracted from `Unitization.lift` below, where it had previously
-been inlined. Unfortunately, `Unitization.lift` was relatively slow in Lean 3, but in Lean 4 it
-just times out. -/
```

Comment:
I agree this definition is nice enough to warrant keeping even if the timeout were fixed.